### ISSUE TITLE
No available provider "jdcloud" plugins are compatible with Terraform version 0.12.0

### ISSUE THREAD

**Issue by RenQilei** [terraform-providers/terraform-provider-jdcloud#7]
I just downloaded the terraform lastest version 0.12.0, which meets 0.10.0+

But when I ran `terraform init`, it shown that no available jdcloud plugins as: 

```
$ terraform init

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...

No available provider "jdcloud" plugins are compatible with this Terraform version.

From time to time, new Terraform major releases can change the requirements for
plugins such that older plugins become incompatible.

Terraform checked all of the plugin versions matching the given constraint:
    (any version)

Unfortunately, none of the suitable versions are compatible with this version
of Terraform. If you have recently upgraded Terraform, it may be necessary to
move to a newer major release of this provider. Alternatively, if you are
attempting to upgrade the provider to a new major version you may need to
also upgrade Terraform to support the new version.

Consult the documentation for this provider for more information on
compatibility between provider versions and Terraform versions.


Error: no available version is compatible with this version of Terraform
```

Is there currently any specific version I have to use if I want to keep using jdcloud.

Thanks.

**Comment by XiaohanLiang** [comment]
Hi Qilei, 

Thank you for your report. We will start inspecting this bug and see what we can do. I believe we have  some job to do in order to make our plugin available in Terraform 0.12

BR,
XiaohanHan

**Comment by XiaohanLiang** [comment]
Hi Qilei,

Sorry for long waiting. After we have updated our plugin, it is now compatible to Terraform0.12.
It might be required to remind you that grammar of `jdcloud.tf`  has slightly changed. We have updated our example. You can check it at our example folder : )

```bash
 ~ $ terraform init

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "jdcloud" (terraform-providers/jdcloud) 1.1.0...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.jdcloud: version = "~> 1.1"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
```

BR,
XiaohanLiang
